.github/workflows/load-testing.yml

```
@@ -0,0 +1,294 @@
+          done
+          
+      - name: Run k6 load test
+        run: |
+          echo "Running k6 load test..."
+          k6 run load-tests/api-load-test.js \
+            --env VUS=${{ github.event.inputs.users || '10' }} \
+            --env DURATION=${{ github.event.inputs.duration || '30s' }} \
+            --out json=load-test-results.json
+        env:
+          OLLAMA_URL: http://localhost:11434
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)